### PR TITLE
[New Rule] Account Password Reset Remotely

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/1570

### CODE REVIEW

File: rules/windows/persistence_remote_password_reset.toml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2021/10/18"
+maturity = "production"
+updated_date = "2021/10/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempt to remotely reset an account. Adversaries may manipulate account password to maintain access or evade
+password duration policies and preserve the life of compromised credentials.
```
Comment (w0rk3r) [inline rules/windows/persistence_remote_password_reset.toml:10]:
```suggestion
Identifies an attempt to reset an account password remotely. Adversaries may manipulate account passwords to maintain
access or evade password duration policies and preserve compromised credentials.
```

File: rules/windows/persistence_remote_password_reset.toml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+"""
+false_positives = ["Legitimate remote accounts administration."]
```
Comment (w0rk3r) [inline rules/windows/persistence_remote_password_reset.toml:12]:
```suggestion
false_positives = ["Legitimate remote account administration."]
```

File: rules/windows/persistence_remote_password_reset.toml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+risk_score = 47
+rule_id = "2820c9c2-bcd7-4d6e-9eba-faf3891ba450"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+sequence by host.id with maxspan=5m
+  [authentication where event.action=="logged-in" and
```
Comment (w0rk3r) [inline rules/windows/persistence_remote_password_reset.toml:32]:
```suggestion
  [authentication where event.action == "logged-in" and
```

File: rules/windows/persistence_remote_password_reset.toml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    /* event 4624 need to be logged */
+    winlog.logon.type:"Network" and event.outcome == "success" and source.ip != null and
```
Comment (w0rk3r) [inline rules/windows/persistence_remote_password_reset.toml:34]:
```suggestion
    winlog.logon.type : "Network" and event.outcome == "success" and source.ip != null and
```

File: rules/windows/persistence_remote_password_reset.toml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an attempt to reset an account password remotely. Adversaries may manipulate account passwords to maintain
+access or evade password duration policies and preserve compromised credentials.
+"""
+false_positives = ["Legitimate remote account administration."]
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*"]
```
Comment (brokensound77) [inline rules/windows/persistence_remote_password_reset.toml:14]: would system integration work as well for this?

File: rules/windows/persistence_remote_password_reset.toml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+sequence by host.id with maxspan=5m
+  [authentication where event.action == "logged-in" and
+    /* event 4624 need to be logged */
+    winlog.logon.type : "Network" and event.outcome == "success" and source.ip != null and
+    source.ip != "127.0.0.1" and source.ip != "::1"] by winlog.event_data.TargetLogonId
```
Comment (brokensound77) [inline rules/windows/persistence_remote_password_reset.toml:35]:
```suggestion
    not source.ip in ("127.0.0.1", "::1")] by winlog.event_data.TargetLogonId
```

**Review by brokensound77 — APPROVED** [review]
Nice rule! Left a question and suggestion then LGTM

File: rules/windows/persistence_remote_password_reset.toml (hunk 5 above)
Comment (w0rk3r) [reply rules/windows/persistence_remote_password_reset.toml:14]: Yes

**Pull request merged**